web/pages/index.tsx

```diff
@@ -2,18 +2,25 @@ const isServer = () => typeof window === 'undefined';
 
 function Index() {
 	if (!isServer()) {
-		import("mixer-cl").then(wasm => {
-			let cl = wasm.Mixer.new([["EDG", 0, 2]]);
-			let note = cl.generate_note("EDG", 0);
+		import("../mixer-client").then(wasm => {
+			let cl = wasm.Mixer.new([["EDG", 0, 32]]);
```

Comment:
can we change this so that it is handled with more descriptive API? Maybe using an object
```
wasm.Mixer.new([{
  asset: 'EDG',
  // not sure what 0 is from looking at it
  depth: 32,
}])
```

Reply:
Sure, that's a good idea